### PR TITLE
change images to src/main/resources

### PR DESCRIPTION
**Pull request by celinepet**

### CODE REVIEW

File: src/main/java/ineat/Bomb.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 public class Bomb extends SpaceObject {
 	
 	public Bomb(int x, int y) {
-		ImageIcon icone = new ImageIcon("images/shot.png");
+		ImageIcon icone = new ImageIcon("src/main/resources/shot.png");
```
Comment (paoue74) [inline src/main/java/ineat/Bomb.java]: I meant, you can add images folder src/main/resources and uses classloader to  get your images.

http://rafaelsteil.com/correctly-loading-resources-images-and-files-in-java/

File: src/test/PlayerTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import junit.framework.TestCase;
+
+public class PlayerTest extends TestCase {
+
+	public void testPlayer() {
+		assetNotNull(new Player());
+	}
+
+	public void testMoveRight() {
```
[the remark this answers is not in the document]
Comment (celinepet) [reply src/test/PlayerTest.java]: Done on branch fix/test

File: src/main/java/com/ineat/Player.java [hunk 3]
```diff
@@ -16,9 +18,10 @@
 	private boolean willMoveLeft;
 	
     public Player() {
-    	ImageIcon icone = new ImageIcon("images/ship.png");
+    	final URL url = getClass().getClassLoader().getResource("ship.png");
+    	ImageIcon icone = new ImageIcon(url);
 		Image image = icone.getImage(); 
-		Image img = image.getScaledInstance(120, 120,  java.awt.Image.SCALE_SMOOTH); 
+		Image img = image.getScaledInstance(120, 120, java.awt.Image.SCALE_SMOOTH);
```
[the remark this answers is not in the document]
Comment (celinepet) [reply src/main/java/com/ineat/Player.java]: done

File: src/main/java/com/ineat/Player.java [hunk 4]
```diff
@@ -16,9 +18,10 @@
 	private boolean willMoveLeft;
 	
     public Player() {
-    	ImageIcon icone = new ImageIcon("images/ship.png");
+    	final URL url = getClass().getClassLoader().getResource("ship.png");
```
[the remark this answers is not in the document]
Comment (celinepet) [reply src/main/java/com/ineat/Player.java]: done

**Pull request merged**